benchmark/utils/preconditioners.hpp

```diff
@@ -22,7 +22,11 @@ DEFINE_string(preconditioners, "none",
               "A comma-separated list of preconditioners to use. "
               "Supported values are: none, jacobi, paric, parict, parilu, "
               "parilut, ic, ilu, paric-isai, parict-isai, parilu-isai, "
-              "parilut-isai, ic-isai, ilu-isai, sor, overhead");
+              "parilut-isai, ic-isai, ilu-isai, sor, overhead"
+#ifdef GINKGO_BUILD_MPI
+              ", schwarz-jacobi, schwarz-ilu, schwarz-ic, schwarz-lu, dist-mg"
```

Comment:
Shouldn't `mg` just be its own thing? You can use it also without mpi.

Reply:
I updated it to use the default options. The default is Gmres+Schwarz-Jacobi as coarse solver for distributed, which may be overkill.